scheduler-utils/src/client/gateway.js

```diff
@@ -24,12 +25,15 @@ const findTransactionTags = (err) => pipe(
 
 function gatewayWith ({ fetch, GRAPHQL_URL }) {
   return async ({ query, variables }) => {
-    return fetch(GRAPHQL_URL, {
-      method: 'POST',
-      headers: { 'Content-Type': 'application/json' },
-      body: JSON.stringify({ query, variables })
-    })
-      .then((res) => res.json())
+    return backoff(
+      () => fetch(GRAPHQL_URL, {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify({ query, variables })
+      })
+        .then(okRes)
+        .then((res) => res.json()),
+      { maxRetries: 2, delay: 300 })
```

Comment:
These values ought to be configurable on `scheduler-utils`. Otherwise, consumers inherit delays from the retry strategy, with no way to circumvent if that strategy is not desired.

Perhaps `scheduler-utils` should accept `GRAPHQL_MAX_RETRIES` and `GRAPHQL_RETRY_BACKOFF` with a default max retry of `0`, so that the retry with backoff is opt-in, also making this a backwards compatible change.